### ISSUE TITLE
panel sliding

### ISSUE THREAD

**Issue by diabloo** [umano/AndroidSlidingUpPanel#566]
hi,
Is there a way to keep the sliding until  the visible part of my slidingpanel get hidden ?

**Comment by amanzan** [comment]
I created a pull request adding this option. See https://github.com/umano/AndroidSlidingUpPanel/pull/577

**Comment by diabloo** [comment]
thank you for your replay , i'am  new to github how can i use your version ?

**Comment by amanzan** [comment]
I don't think you can use it just by adding dependencies until my pull request is reviewed and merged...you would probably have to download the library, add my changes, and add the modified library to your project. Maybe there's an easier way...but that's how I did it. If you need help doing that, I can try to help you!

**Comment by h4h13** [comment]
inside `onPanelSlide` 
```if (slideOffset >= 0.5) {
       mPauseSong.setAlpha(slideOffset * 0);
     } else {
       mPauseSong.setAlpha(slideOffset + 1);
 }```
